### ISSUE TITLE
About HOTA

### ISSUE THREAD

**Issue by FangLintao** [cheind/py-motmetrics#151]
I am using py-motmetrics super well but in mot-challenge, it has introduced the HOTA metric.
So may I ask when the HOTA will also include in this library?

Thanks

**Comment by cheind** [comment]
@FangLintao , @wenbodu971228 I agree, adding HOTA would be a nice addon. I'd be happy to include a PR if someone finds the time.

Best,
Christoph

**Comment by chenrxi** [comment]
same

**Comment by Yrij-Zhavoronkov** [comment]
Where is HOTA?)

**Comment by andreamaral99** [comment]
Will HOTA be added to this library?

**Comment by johnnynunez** [comment]
any news?

**Comment by folkien** [comment]
Joining HOTA waitlist.

**Comment by mikel-brostrom** [comment]
+1, would be nice to be able to use `py-motmetrics` instead of `trackeval`

**Comment by nathanlem1** [comment]
HOTA is still missing in this nice evaluation tool. It would be nice to include it.

**Comment by mikel-brostrom** [comment]
For those who wonder and want to give it a try:

Official HOTA in `trackeval`:

https://github.com/JonathonLuiten/TrackEval/blob/12c8791b303e0a0b50f753af204249e622d0281a/trackeval/metrics/hota.py#L9

Should go here:

https://github.com/cheind/py-motmetrics/blob/beb864a56d6055047c4d6e015188fcc24aca05b7/motmetrics/metrics.py
